libraries/ms-common/rules.mk

```diff
@@ -8,5 +8,5 @@
 $(T)_DEPS := $(PLATFORM_LIB) libcore
 
 ifeq (x86,$(PLATFORM))
-$(T)_EXCLUDE_TESTS := can can_hw adc
+$(T)_EXCLUDE_TESTS := can can_hw adc pwm
```

Comment:
I'll look into ADC and PWM drivers on Linux if I have time but I'm not sure if there is a nice way to emulate it. IMO SocketCAN on x86 has lots of value but PWM and ADC drivers less so.